### ISSUE TITLE
[Bug] Patient with Serious Radiation effect got Corrugated Ankles

### ISSUE THREAD

**Issue by Matroftt** [CorsixTH/CorsixTH#2323]
### Describe the issue

So, I was playing and noticed this. Also, before it I saw patient with serious radiation disease, but without effect.

### Steps to Reproduce

1. *Load save game attached to this ticket*

2. You can see this patient

### Expected Behaviour

Patient with Corrugated Ankles shouldn't get this effect

### Save Game

Save game attached to report: Yes

https://drive.google.com/file/d/1ZPRvYK2Yr2L9Uw2v0XL-oRnklXPkqDwg/view?usp=sharing

### CorsixTH Version

0.66

### Operating System

Windows 7

### Theme Hospital Version

GOG.com

### Gamelog.txt

```bash
---------------------------------------------------------------

Welcome to CorsixTH v0.66!

---------------------------------------------------------------

This window will display useful information if an error occurs.

---------------------------------------------------------------

Checking for CorsixTH updates...
You are running the latest version of CorsixTH.
Savegame version is 170 (v0.66), originally it was 170 (v0.66).
-----------------------------------
Clicked on:
humanoid - class: Standard Female Patient
Warmth: 0.284   Happiness: 1.000   Fatigue: 0.000  Thirst: 0.645  Toilet_Need: 0.431   Health: 0.718   Service Quality: 0.000
Actions: [use_object - bench   must_happen, queue - Bench distance: 2 Standing: false   must_happen, walk - going to 53:91]
-----------------------------------
```

**Comment by lewri** [comment]
Possibly fixed by #2260 ? @SilPho would that PR resolve do you know?

**Comment by SilPho** [comment]
Only contagious diseases can be overwritten by other contagious diseases. #2260 changes Serious Radiation to be non-contagious, so yes, it's very likely that it would have prevented this issue from happening
